packages/assistify-ai/client/views/app/tabbar/smarti.js

```diff
@@ -45,46 +45,34 @@ Template.AssistifySmarti.onRendered(function() {
 			}
 		} else {
 			instance.smartiLoaded.set(true);
-			const DBS_AI_Redlink_URL =
-				RocketChat.settings.get('DBS_AI_Redlink_URL').endsWith('/') ?
-					RocketChat.settings.get('DBS_AI_Redlink_URL') :
-					`${ RocketChat.settings.get('DBS_AI_Redlink_URL') }/`;
-
-			const SITE_URL_W_SLASH =
-				RocketChat.settings.get('Site_Url').endsWith('/') ?
-					RocketChat.settings.get('Site_Url') :
-					`${ RocketChat.settings.get('Site_Url') }/`;
-
+			const SMARTI_URL = RocketChat.settings.get('DBS_AI_Smarti_URL').replace(/\/?$/, '/');
+			const ROCKET_CHAT_URL = RocketChat.settings.get('Site_Url').replace(/\/?$/, '/');
 			// stripping only the protocol ("http") from the site-url either creates a secure or an insecure websocket connection
-			const WEBSOCKET_URL = `ws${ SITE_URL_W_SLASH.substring(4) }websocket/`;
-
-			let customSuffix = RocketChat.settings.get('Assistify_AI_DBSearch_Suffix') || '';
-			customSuffix = customSuffix.replace(/\r\n|\r|\n/g, '');
-
+			const WEBSOCKET_URL = `ws${ ROCKET_CHAT_URL.substring(4) }websocket/`;
 			const WIDGET_POSTING_TYPE = RocketChat.settings.get('Assistify_AI_Widget_Posting_Type') || 'postRichText';
-			console.log(WIDGET_POSTING_TYPE, RocketChat.settings.get('Assistify_AI_Widget_Posting_Type'));
-
+			const SMARTI_API_PROXY = `${ ROCKET_CHAT_URL }api/v1/assistify/smarti/`;
 			const smartiOptions = {
 				socketEndpoint: WEBSOCKET_URL,
-				smartiEndpoint: DBS_AI_Redlink_URL,
+				smartiEndpoint: SMARTI_URL,
+				rocketBaseurl: ROCKET_CHAT_URL,
+				smartiApiProxy: SMARTI_API_PROXY,
 				channel: instance.data.rid,
 				postings: {
 					type: WIDGET_POSTING_TYPE,
 					cssInputSelector: '.rc-message-box .js-input-message'
 				},
 				widget: {
 					'query.dbsearch': {
-						numOfRows: 2,
-						suffix: customSuffix
+						numOfRows: 2
 					},
 					'query.dbsearch.keyword': {
 						numOfRows: 2,
-						suffix: customSuffix,
 						disabled: true
 					}
 				},
```

Comment:
Since the Solr search has moved to the client configuration in Smarti, I think we could remove the whole widget options.